python/servo/command_base.py

```diff
@@ -51,6 +51,10 @@ class CommandBase(object):
     def __init__(self, context):
         self.context = context
 
+        def resolverelative(category, key):
+            if self.config[category][key].startswith("./"):
```

Comment:
You want to call [`os.path.expanduser`](https://docs.python.org/2/library/os.path.html#os.path.expanduser) first.